crates/autopilot/src/driver_api.rs

```diff
@@ -0,0 +1,45 @@
+use crate::driver_model::{execute, solve};
```

Comment:
ubernit:
I actually think that both the data-model and the HTTP interface can live in the same `driver_api` (or even just `driver`) module within the autopilot WDYT?

Reply:
If the model only lives in the autpilot crate it doesn't matter. If we move it to a shared crate then could make sense to split it up for example because driver implementation of the API wants to reuse the model (I know we might want to do that). Could also want to reuse the model for the orderbook calling /quote on drivers without the same http implementation.

I'd keep it split for easier refactoring later, for now.